packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -14,13 +14,13 @@ import {
   Item,
 } from '@openmsupply-client/common';
 import { reducer, OutboundAction, itemToSummaryItem } from './reducer';
-import { getOutboundShipmentDetailViewApi } from '../../api';
+import { getOutboundShipmentDetailViewApi } from './api';
 import { GeneralTab } from './tabs/GeneralTab';
 import { ItemDetailsModal } from './modals/ItemDetailsModal';
 
-import { OutboundShipmentSummaryItem } from './types';
+import { OutboundShipmentSummaryItem } from '../../types';
 import { Toolbar } from './Toolbar';
-import { isInvoiceEditable } from '../utils';
+import { isInvoiceEditable } from '../../utils';
```

Comment:
moved `Invoices/Outbound/utils`  -> `Invoices/utils` to use them in `Invioce/Inbound` also

Reply:
Same with types